.github/steps/5-trigger.md

```diff
@@ -18,6 +18,7 @@ Your new action will run any time a pull request has been opened.
 
 1. Make a new branch named `test-workflow`.
 1. Commit any change to your branch, such as adding an emoji to your README.md file.
+1. Validate that your GITHUB_TOKEN has write permission on your repository. (Go to Settings->Actions->General->Workflow permissions)
```

Comment:
I'd suggest changing this slightly, to avoid reference to GITHUB_TOKEN at this stage:

```suggestion
1. Click the **Settings** tab for your repository. In the left panel, click **Actions** then **General**. Under "Workflow permissions," make sure that **Read and write permissions** is selected. This allows your workflows to write to this repository.
1. Make a new branch named `test-workflow`.
1. Commit any change to your branch, such as adding an emoji to your README.md file.
```